src/site/content/en/blog/preload-scanner/index.md

```diff
@@ -0,0 +1,285 @@
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/irM6kTXZoP0MmXGtHOSa.jpg
+alt: A close-up photograph of a screen with various bits of HTML markup shown in a text editor.
+description: |
+  Find out what the browser preload scanner is, how it helps performance, and how you can stay out of its way.
+tags:
+  - blog
+  - performance
+  - web-vitals
+---
+
+One overlooked aspect of optimizing page speed involves knowing a bit about browser internals. Browsers make certain optimizations for us to improve performance in ways that we as developers can't—but only so long as we don't thwart those optimizations unintentionally.
+
+One internal browser optimization to understand is the browser preload scanner. In this post, we'll talk a bit about how the preload scanner works—and more importantly, how you can avoid getting in its way.
+
+## What's a preload scanner?
+
+Every browser has a primary HTML parser that [tokenizes](https://en.wikipedia.org/wiki/Lexical_analysis#Tokenization) raw markup and processes it into [an object model](https://developer.mozilla.org/docs/Web/API/Document_Object_Model). This all merrily goes on until the parser pauses when it finds a [blocking resource](/render-blocking-resources/), such as a stylesheet loaded with a `<link>` element, or script loaded with a `<script>` element without an `async` or `defer` attribute.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/mXRoJneD6CbMAqaTqNZW.svg", alt="A diagram of how the browser's primary HTML parser parses and tokenizes markup into the DOM. The parser is blocked from doing any further work when it encounters a &lt;link&gt; element pointing to a stylesheet, and can only resume its work until the stylesheet is downloaded and processed.", width="111", height="150" %}
+  <figcaption>
+    <strong>Fig. 1:</strong> A diagram of how the browser's primary HTML parser can be blocked. In this case, the parser runs into a <code>&lt;link&gt;</code> element for an external CSS file, which blocks the browser from parsing the rest of the document—or even rendering any of it—until the CSS is downloaded and parsed.
+  </figcaption>
+</figure>
+
+In the case of CSS files, both parsing and rendering are blocked in order to prevent a [flash of unstyled content (FOUC)](https://en.wikipedia.org/wiki/Flash_of_unstyled_content), which is when an unstyled version of a page can be seen briefly before styles are applied to it.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/8LkMd46pUlwgfYvmTBrO.png", alt="A comparison of the web.dev home page in an unstyled state (left) and the styled state (right). The unstyled state can occur in a flash if the browser doesn't block rendering while a stylesheet is being downloaded and processed.", width="800", height="748", loading="lazy" %}
+  <figcaption>
+    <strong>Fig. 2:</strong> A simulated example of FOUC. At left is the front page of web.dev without styles. At right is the same page with styles applied.
+  </figcaption>
+</figure>
```

Comment:
```suggestion
<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/8LkMd46pUlwgfYvmTBrO.png", alt="The web.dev home page in an unstyled state (left) and the styled state (right).", width="800", height="748", loading="lazy" %}
  <figcaption>
    <strong>Fig. 2:</strong> A simulated example of FOUC. At left is the front page of web.dev without styles. At right is the same page with styles applied. The unstyled state can occur in a flash if the browser doesn't block rendering while a stylesheet is being downloaded and processed.
  </figcaption>
</figure>
```

Reply:
That flash detail is interesting and should be seen by all in the caption!

Also, another a11y related alt text note. Some screenreaders have a character limit for alt text of 150, so the shorter the better.